compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Window.desktop.kt

```diff
@@ -187,6 +191,19 @@ fun Window(
                         appliedState.position = currentState.position
                     }
                 })
+                when (hostOs) {
+                    OS.MacOS -> {
+                        addComponentListener(object : ComponentAdapter() {
+                            override fun componentShown(e: ComponentEvent) {
+                                if (initialPlacement != placement) {
+                                    placement = initialPlacement
```

Comment:
The bug is in skiko. We should make a change in skiko, as the property `SkiaLayer.isFullscreen` breaks its contract:
- set window to fullscreen, if we change it to true
- return `isFullscreen=true` if we set it to true, if even the window is not displayable yet (we don't do that, as we don't store intermideate stae, we just asynchronously send setFullscreen to appKit thread)